### PR TITLE
Find repeating and missing number in array using CPP

### PR DESCRIPTION
**Pull request by shahina-bano**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added the solution for the problem in CPP

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

The issue is #5116

### CODE REVIEW

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Shift the logic to another function and use main for driver code

**Comment by shahina-bano** [comment]
> Shift the logic to another function and use main for driver code

I have done the changes, is it okay? Please let me know for any further changes @ankitaggarwal23

File: C-Plus-Plus/other/repeated_missing.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            cout << "Missing number: " << i+1 << endl;
+        }
+    }
+}
+
+int main() {
+    // input the size from user
+    int n;
+    cin >> n;
+    //input the array elements
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/repeated_missing.cpp]: Remove the redundant comments instead write proper display message using `cout`

File: C-Plus-Plus/other/repeated_missing.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    int input[n];
+    for(int i = 0; i < n; i++)
+        cin >> input[i];
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/repeated_missing.cpp:42]: write proper display message using `cout`

File: C-Plus-Plus/other/repeated_missing.cpp [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Repeated number: 2
+Missing number: 3
+
+Test Case 2:
+Input - 
+7
+4 2 3 6 1 1 7
+Output - 
+Repeated number: 1
+Missing number: 5
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/repeated_missing.cpp]: Remove extra test case one is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/README.md [hunk 4]
```diff
@@ -522,3 +522,4 @@ _add list here_
 - [Activity Selection](other/ActivitySelection.cpp)
 - [Job sheduling with given deadlines](other/job_sheduling_with_deadlines.cpp)
 - [Union and Intersection of arrays](other/Union_and_Intersection_of_arrays.cpp)
+- [Find repeating and missing number in array](other/repeated_missing.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Rewrite this ` [Find repeating and missing number in array]` to like that `[Repeating and Missing Number]`

File: C-Plus-Plus/other/repeated_missing.cpp [hunk 5]
```diff
@@ -0,0 +1,61 @@
+/*
+Repeating and missing number
+Problem Statement: Given an unsorted array of positive integers. The elements are in the range 1 to n where n is the size of array,
+                   there is one number which is repeated and another number which is missing. Find these numbers and print them.
+		    The approach used is that checking if the current element is in its desired position or not. It is, then simply iterate
+		    for next, else put it in its desired position by comparing with its index number and swap it accordingly.
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/repeated_missing.cpp]:
```
/*
Repeating and Missing number

Given an unsorted array of positive integers. The elements are in the range 1 to n where n is the size of array,
there is one number which is repeated and another number which is missing. Find these numbers and print them.
The approach used is that checking if the current element is in its desired position or not. It is, then simply iterate
for next, else put it in its desired position by comparing with its index number and swap it accordingly.

*/
```

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
LGTM.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**